emailer/src/emailer.js

```diff
@@ -27,7 +32,54 @@ const getInactiveApplications = async () => {
 
   await client.end()
 
-  return result
+  return result.rows
+}
+
+try {
+  let root, mailConfig
+  if (productionEnv) {
+    root = process.env.SITE_URL
+    mailConfig = {
+      host: 'smtp.sendgrid.net',
+      port: 587,
+      pool: true,
+      secure: false,
+      auth: {
+        user: 'apikey',
+        pass: process.env.SENDGRID_APIKEY
+      }
+    }
+  } else {
+    root = 'http://localhost:3000'
+    mailConfig = {
+      //logger: true,
+      host: 'smtp.ethereal.email',
+      port: 587,
+      auth: {
+        user: process.env.ETHEREAL_USERNAME,
+        pass: process.env.ETHEREAL_PASSWORD
+      }
+    }
+  }
+
+  const emailAddress = 'Recruitment Tool <careers@emblica.com>'
+  let transporter = nodemailer.createTransport(mailConfig, { to: emailAddress, from: emailAddress })
+
+  const inactiveApplications = getInactiveApplications()
+  inactiveApplications.then(res => res.map((application) => {
```

Comment:
Variable `inactiveApplications` could be left out, as it's not needed anywhere else. This would then be simply `getInactiveApplications().then(...`